docker/template.conf

```diff
@@ -0,0 +1,105 @@
+include "metadata.conf"
+
+conseil: {
+  hostname: "0.0.0.0"
+  port: {{APIPORT}}
+  cache-ttl: 15 minutes
+  max-query-result-size: 100000
+}
+
+lorre: {
+  sleepInterval: 5 s
+  bootupRetryInterval: 10 s
+  bootupConnectionCheckTimeout: 10 s
+  #Used to make sure Lorre records average fees every n iterations
+  feeUpdateInterval: 20
+  #Used to select how many fees should be averaged together
+  numberOfFeesAveraged: 1000
+  depth: newest,
+  headHash: None
+}
+
+conseildb = {
+  dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
+  properties = {
+    databaseName = "{{DBDB}}"
+    user = "{{DBUSER}}"
+    password = "{{DBPW}}"
+    url = "jdbc:postgresql://{{DBHOST}}:{{DBPORT}}/{{DBDB}}"
+  }
+  numThreads = 64
```

Comment:
Maybe on a later PR:
these values are custom-tailored based on specific environmental resources.
The number is quite high, and there's an heuristic to compute it based on cores and disk access speed.

It might be worthwhile to inject these via envvars too (even using a single var, as they should match anyway), enabling custom tailoring based on measured usage of db resources and jvm threads

Reply:
Or we could simply leave this to the use of the custom configuration file override...